picard/tagger.py

```diff
@@ -143,9 +143,7 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
         if not IS_MACOS and not IS_HAIKU:
             self.setStyle('Fusion')
 
-        # Set the WM_CLASS to 'MusicBrainz-Picard' so desktop environments
```

Comment:
Is this comment no longer applicable?

Reply:
No, the relevant code was removed. Turns out this workaround was actually only necessary because we did not provide the full argv (with argv[0] being set to the application path) to Qt